test/data.c

```diff
@@ -306,6 +306,7 @@ bool bmautoadd = false;
 bool bmautograb = false;
 
 /*-------------------------------------rigctl-------------------------------*/
+char rotconf[80];
 #ifdef HAVE_LIBHAMLIB
```

Comment:
Hamlib is mandatory, please drop both checks. It just hasn't been cleaned up.

Reply:
Updated,